### ISSUE TITLE
Improve face alignment using a better align-dlib.py 

### ISSUE THREAD

**Issue by delip** [davidsandberg/facenet#19]
Per this [new discussion](https://groups.google.com/forum/?utm_medium=email&utm_source=footer#!msg/cmu-openface/aQuYBLcVKAo/ZBC5mLaLAQAJ) on OpenFace, I think your models would benefit from a better dlib aligner. This is a drop in replacement for existing align-dlib.py. Checkout the new  [align-dlib.py.zip](https://github.com/davidsandberg/facenet/files/278441/align-dlib.py.zip)

**Comment by davidsandberg** [comment]
Very interesting thread! Definitely worth looking into. Thanks!

**Comment by delip** [comment]
FYI, this new alignment bumps accuracy from  0.9292 to 0.9425 on LFW

**Comment by davidsandberg** [comment]
That's a very good result! Did you just apply the new alignment to the LFW test set, or did you apply it to the training set and retrain as well?

**Comment by delip** [comment]
You need to retrain with the new alignments, and also apply to the test set. I would imagine not using the same alignment in test and train would make the results worse.

**Comment by delip** [comment]
Also, I did not do this experiment. This result is from Brandon @ OpenFace. I want to retrain the models to try out a bunch of ideas, but I'm not easily getting access to CASIA (I have Facescrub).

**Comment by delip** [comment]
Good news. I got access to CASIA dataset!
